phpBB/styles/prosilver/theme/content.css

```diff
@@ -557,6 +557,11 @@ blockquote .codebox {
 	padding: 5px 3px;
 }
 
+/* Mention block */
+.mention {
```

Comment:
where is this used?

Reply:
This is used in posts for handling the parsed mention BBCode.